### ISSUE TITLE
Is there any plan to release this to pypi?

### ISSUE THREAD

**Issue by fabianvf** [aneesha/RAKE#13]
This would be a great package to have in pypi, especially if it were python3 compatible. Is the project still being actively maintained? Are you looking for a maintainer? I would be happy to help any way I can.

**Comment by pmbaumgartner** [comment]
@fabianvf I've been working on a rewrite to clean things up, add functionality, and likely release to pypi. You seem to be the most active on this package, would you mind taking a look at my rewrite and let me know what you think? https://github.com/pmbaumgartner/RAKE

**Comment by fabianvf** [comment]
Absolutely. Could you submit a pull request to my fork so I can review the changes there?
